src/vivarium_public_health/risks/distributions.py

```diff
@@ -294,10 +350,46 @@ def ppf(self, quantiles: pd.Series) -> pd.Series:
 
 
 class DichotomousDistribution(RiskExposureDistribution):
+
     #################
     # Setup methods #
     #################
 
+    def get_exposure_value_columns(self, builder: Builder) -> Optional[List[str]]:
+        if isinstance(self.exposure_data, pd.DataFrame):
+            return builder.data.value_columns()(self.exposure_data)
+        return None
+
+    def get_exposure_data(self, builder: Builder) -> Union[int, float, pd.DataFrame]:
+        exposure_data = super().get_exposure_data(builder)
+
+        if isinstance(exposure_data, (int, float)):
+            return exposure_data
+
+        # rebin exposure categories
+        self.validate_rebin_source(builder, exposure_data)
+        rebin_exposed_categories = set(self.configuration["rebinned_exposed"])
```

Comment:
Would self.configuration ever not have a "rebinned_exposed" key at this point?

Reply:
It can't not since, since there is a configuration default at the component layer of an empty list.